spec/rubocop/cop/rspec/rails/have_http_status_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Rails::HaveHttpStatus do
+  it 'registers an offense for `expect(response.status).to be 200`' do
+    expect_offense(<<~RUBY)
+      it { expect(response.status).to be 200 }
+           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `expect(response.status).to be 200` over `expect(response).to have_http_status(200)`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      it { expect(response).to have_http_status 200 }
```

Comment:
This is subjective, but I'd prefer it to have parentheses `have_http_status(200)`.

Reply:
Fixed in https://github.com/rubocop/rubocop-rspec/pull/1307/commits/6cd44545c4e06741bcd8315fd7f520ca5be898b8